tests/test_pipelines.py

```diff
@@ -82,6 +82,32 @@
     "truth_summary",
 }
 
+# LSSTCam common inputs, in addition to COMMON_INPUTS
+LSSTCAM_INPUTS = {
+    "bfk",
+    "linearizer",
+    "defects",
+    "the_monster_20240904",
+    "gaia_dr3_20230707",
```

Comment:
I think that the astrometric refcat should be the monster as well.

Reply:
I wasn't sure if it was time to make that switch but I'm happy to do it.

Reply:
I just merged the fix to meas_algorithms so it's good to go.  Also we don't have gaia_dr3 on the summit.